docs: npx expo customize tsconfig.json (#24170)
expo/expo: docs/pages/router/reference/typed-routes.mdx

Expo Router supports generating TypeScript types automatically with Expo CLI. This enables `<Link>`, and the [hooks API](/router/reference/hooks) to be statically typed. This feature is currently in beta and is not enabled by default.

> ## Getting started

Reviewer: ```suggestion
## Get started
```

Let's use short form titles instead of gerunds.

}
```

Run `npx expo customize tsconfig.json` to configure your **tsconfig.json** file. This will add the required `includes` fields to your **tsconfig.json**.

> Start the development server by running `npx expo start` and you can now use autocomplete in the Expo Router `<Link />` component's `href` prop.

Reviewer: ```suggestion
Start the development server by running `npx expo start`. You can now use autocomplete in the Expo Router `<Link>` component's `href` prop.
```

}
```

> Run `npx expo customize tsconfig.json` to configure your **tsconfig.json** file. This adds the required `includes` fields to the **tsconfig.json**.

Reviewer: ```suggestion
Run `npx expo customize tsconfig.json` to configure your **tsconfig.json** to add the required `includes` fields.
```

> Start the development server by running `npx expo start`. You can now use autocomplete in the Expo Router `<Link>` component's `href` prop.

Reviewer: ```suggestion
Then, start the development server by running `npx expo start`. You can now use autocomplete in the Expo Router `<Link>` component's `href` prop.
```

> ## Type Generation

Reviewer: ```suggestion
## Type generation
```

> Typed routes in Expo Router are automatically generated when the development server starts. By default, these generated types are configured to be untracked by Git and will be added to a local **.gitignore** file. This ensures that autogenerated files do not clutter your version control system. 

Reviewer: ```suggestion
Typed routes in Expo Router are automatically generated when the development server starts. By default, these generated types are configured to be untracked by Git and will be added to the local **.gitignore** file. This ensures that autogenerated files do not clutter your version control system. 
```
